testdata/rag/README.md

```
@@ -0,0 +1,203 @@
+
+The script uses HTTP APIs only — no `docker compose exec` — so it
+works with Docker, Podman, or any setup where the services are
+reachable. Override the defaults with environment variables:
+
+```bash
+WEAVIATE_URL=http://localhost:8080 OLLAMA_URL=http://localhost:11434 ./seed.sh
+```
+
+Expected output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)